### PR TITLE
Adios threading 1

### PR DESCRIPTION
**Pull request by tkurc**
Added threading=1 to use separate thread for I/O operations.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,8 @@ option (PIO_USE_MALLOC       "Use native malloc (instead of bget package)"  OFF)
 option (PIO_MICRO_TIMING     "Enable internal micro timers"                 OFF)
 option (PIO_SAVE_DECOMPS     "Dump the decomposition information"           OFF)
 option (WITH_PNETCDF         "Require the use of PnetCDF"                   ON)
-option (WITH_NETCDF          "Require the use of NetCDF"                    ON)
-option (WITH_ADIOS           "Require the use of ADIOS"                     OFF)
+option (WITH_NETCDF          "Require the use of NetCDF"                    OFF)
+option (WITH_ADIOS           "Require the use of ADIOS"                     ON)
```
Comment (jayeshkrishna) [inline CMakeLists.txt]: These defaults shouldn't be changed. You can change it to test ADIOS by setting it at configure time.

**Comment by jayeshkrishna** [comment]
Also I see a lot of binary files checked in the commits. Can you remove them?
You can rebase the branch with just the required changes (the only file to be changed is pioc_support.c)

File: CMakeLists.txt (hunk 1 above)
Comment (tkurc) [reply CMakeLists.txt]: Done. Will create a new pull request.

**Comment by jayeshkrishna** [comment]
A newer version of this PR is #235 . Closing this one and continuing the integration in PR #235 .

**Pull request closed without merging**